sdk/golang/ccnp/quote/quote.go

```diff
@@ -0,0 +1,191 @@
+/*
+* Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+* SPDX-License-Identifier: Apache-2.0
+ */
+
+package quote
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/binary"
+	"log"
+	"strings"
+	"time"
+
+	pb "github.com/intel/confidential-cloud-native-primitives/sdk/golang/ccnp/quote/proto"
+	pkgerrors "github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+const (
+	UDS_PATH = "unix:/run/ccnp/uds/quote-server.sock"
+	TYPE_TDX = "TDX"
+	TYPE_TPM = "TPM"
+)
+
+type TPMQuote struct{}
+
+type TDXQuote struct {
+	Quote           []uint8    // full TD quote
+	Version         uint16     // TD quote version
+	Tdreport        [584]uint8 // full TD report
+	Tee_type        uint32     // Type of TEE for which the Quote has been generated
+	Tee_tcb_svn     [16]uint8  // Array of TEE TCB SVNs
+	Mrseam          [48]uint8  // Measurement of the SEAM module (SHA384 hash)
+	Mrsignerseam    [48]uint8  // Measurement of a 3rd party SEAM module’s signer (SHA384 hash)
```

Comment:
And in the spec, it is called MRSEAMSIGNER.

Reply:
updated